src/library/scala/collection/concurrent/TrieMap.scala

```diff
@@ -943,12 +943,19 @@ extends scala.collection.concurrent.Map[K, V]
 
   override def stringPrefix = "TrieMap"
 
+  // This is needed to instantiate the updater in this class, because it is accessing
+  // a private field, and then leak it to the companion object for storage
+  private def updater(guard: TrieMap.UpdaterGuard): AtomicReferenceFieldUpdater[TrieMap[_, _], AnyRef] = {
```

Comment:
This doesn't work anyway because TrieMap.UpdaterGuard is public when viewed from Java.